mosh/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_name=mosh
+pkg_origin=core
+pkg_version=1.2.5
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('GPL-3.0')
+pkg_description="Remote Terminal application that allows roaming, supports intermittent connectivity, and provides intelligent local echo and lines
+editing of users keystrokes"
+pkg_upstream_url="https://mosh.mit.edu"
+pkg_source=https://mosh.mit.edu/mosh-${pkg_version}.tar.gz
+pkg_shasum=1af809e5d747c333a852fbf7acdbf4d354dc4bbc2839e3afe5cf798190074be3
+pkg_build_deps=(core/coreutils core/patch core/autoconf core/automake core/gcc core/make core/glibc core/protobuf core/zlib core/openssl core/pkg-config core/ncurses)
```

Comment:
I think it's better, if you have more than a few of these, to put them each on one line, alphabetically. Makes it easier to add and remove, as well as read.